ha/util/conf_store.py

```diff
@@ -115,6 +115,8 @@ def set_cluster_cardinality(self, index):
         cluster_cardinality_value = {CLUSTER_CARDINALITY_NUM_NODES: num_pods, CLUSTER_CARDINALITY_LIST_NODES : watch_pods }
         self._confstore.update(cluster_cardinality_key, json.dumps(cluster_cardinality_value))
 
+        return data_pods, server_pods, control_pods, num_pods, watch_pods
```

Comment:
We don't need to return "num_pods" from here as it only returns the length of "watch_pods".
Also this is unused variable in _add_node_health() method.